tests/TestHarness/testUtils.py

```diff
@@ -76,8 +76,8 @@ class Utils:
     DataRoot=os.path.basename(sys.argv[0]).rsplit('.',maxsplit=1)[0]
     PID = os.getpid()
     DataPath= f"{TestLogRoot}/{DataRoot}{PID}"
-    DataDir= f"{DataPath}/"
-    ConfigDir=f"{str(Path.cwd().resolve())}/etc/eosio/"
+    DataDir=f"{DataPath}/"
+    ConfigDir=f"{DataPath}/"
```

Comment:
I waffled on this.  There are a dozen references to a config directory still which includes minor amounts of external usage.  I'm not completely sanguine about the prospect of those two paths remaining the same, since in practice there's now only five tests where a developer needs to look inside it when examining a problem.  With nearly all configuration done on the command line, the configuration files written by nodeos are just cruft for all but those five tests, so I could see being asked to bury them all in a separate hierarchy.  I left it for the time being.

Reply:
That's fine.  Just caught my eye and thought I'd ask.